README.md

```diff
@@ -43,6 +43,18 @@ To upgrade to the latest (unstable) version, run
 pip install --upgrade git+https://github.com/cornellius-gp/gpytorch.git
 ```
 
+### ArchLinux Package
+Note: The ArchLinux package is not tested by the GPyTorch developers.
```

Comment:
Instead of this, let's say `Note: Experimental AUR package. For most users, we recommend installation by conda or pip.`

Reply:
Sounds good.